calc.js

```diff
@@ -1,22 +1,17 @@
-var numField1 = document.getElementById('numField1');
-var numField2 = document.getElementById('numField2');
+var numerator = document.getElementById('numField1');
+var denominator = document.getElementById('numField2');
```

Comment:
these names make it easier to know where the values should end up in the calculation